src/main/kotlin/com/openlattice/chronicle/controllers/CandidatesController.kt

```diff
@@ -41,6 +46,32 @@ class CandidatesController @Inject constructor(
     @Inject
     private lateinit var candidatesService: CandidatesService
 
+    @Timed
+    @GetMapping(
+        path = [CANDIDATE_ID_PATH],
+        produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    override fun getCandidate(@PathVariable(CANDIDATE_ID_PARAM) candidateId: UUID): Candidate {
+        ensureAuthenticated()
```

Comment:
Candidates should be only be readable by themselves. Perhaps check that e-mails match?

Reply:
First, there's no email to check against. Yes, we can add an `email` column to the `candidates` table; please confirm you want me to do that.

Second, checking if emails match would mean no users will be able to call this endpoint. In other words, researcher emails would never match candidate emails so researchers would never be able to call this api. Is that the behavior you're looking for?

Third, candidates don't have accounts yet.

Reply:
This is accurate. Participants for study should be listed via the study API. A candidate should be created for each study participant, but not all candidate fields may get collected when that happens.